.clang-format

```diff
@@ -0,0 +1,3 @@
+BasedOnStyle: Google
+IndentWidth: 4
+ColumnLimit: 120
```

Comment:
Nice, thanks for this change!

Reply:
This has one nice property, it allows one to call "clang-format" anywhere in the tree, and just get something consistent across the three and, from my point of view, nice looking.